official/nlp/modeling/ops/beam_search.py

```diff
@@ -409,7 +412,7 @@ def _create_initial_state(self, initial_ids, initial_cache, batch_size):
     alive_seq = expand_to_beam_size(initial_ids, self.beam_size)
     alive_seq = tf.expand_dims(alive_seq, axis=2)
     if self.padded_decode:
-      alive_seq = tf.tile(alive_seq, [1, 1, self.max_decode_length + 1])
+        alive_seq = tf.pad(alive_seq, [[0, 0], [0, 0], [0, self.max_decode_length]], constant_values=self.pad_id)
```

Comment:
We need to remove the extra spaces in front of this line to conform to our Python styling

Reply:
I think need to resolve the conflict as well, will create a new PR with all above changes.